base/src/ImageEncoderCV.cpp

```diff
@@ -142,8 +152,10 @@ bool ImageEncoderCV::process(frame_container &frames)
 	{
 		return true;
 	}
+	// mDetail->executeCallback();
```

Comment:
clean

Reply:
Cleaned, Added Error Callback in GtkGlRenderer module